### PR TITLE
Fixed auto after testing on robot

### PR DESCRIPTION
**Pull request by ThinkRedstone**

### CODE REVIEW

File: src/com/spikes2212/robot/commands/autonomous/MoveStraightAndDropGearAuto.java [hunk 1]
```diff
@@ -14,14 +17,24 @@
...
 			.addConstantDouble("MoveStraightAndDropGearAuto-movingTime", 3);
+	public static final Supplier<Double> afterOrientionTime = ConstantHandler
+			.addConstantDouble("MoveStraightAndDropGearAuto-afterOrientionTime", 3);
+	public static final Supplier<Double> reverseTime = ConstantHandler
+			.addConstantDouble("MoveStraightAndDropGearAuto-reverseTime", 1.5);
+	public static final Supplier<Double> reverseSpeed = ConstantHandler
+			.addConstantDouble("MoveStraightAndDropGearAuto-reverseSpeed", 0.3);
 
 	public MoveStraightAndDropGearAuto() {
 		addSequential(new DriveTank(Robot.drivetrain, movingSpeed, movingSpeed), movingTime.get());
```
Comment (TomerSitton) [inline src/com/spikes2212/robot/commands/autonomous/MoveStraightAndDropGearAuto.java:33]: please make this one DriveArcade as well. Or make the other one DriveTank

File: src/com/spikes2212/robot/commands/autonomous/MoveTurnLeftAndDropGearAuto.java [hunk 2]
```diff
@@ -16,18 +14,17 @@
...
-	public static final Supplier<Double> moveToGearTime = ConstantHandler
-			.addConstantDouble("MoveTurnLeftAndDropGearAuto-moveToGearTime", 1);
-
 	public MoveTurnLeftAndDropGearAuto() {
-		addSequential(new DriveTank(Robot.drivetrain, MoveStraightAndDropGearAuto.movingSpeed,
-				MoveStraightAndDropGearAuto.movingSpeed), movingStraightTime.get());
-		addSequential(new OrientateAndMoveToGear(turningSpeed, MoveStraightAndDropGearAuto.movingSpeed),
-				moveToGearTime.get());
+		addSequential(
+				new DriveTank(Robot.drivetrain, MoveStraightAndDropGearAuto.movingSpeed,
```
Comment (TomerSitton) [inline src/com/spikes2212/robot/commands/autonomous/MoveTurnLeftAndDropGearAuto.java:19]: please make this one driveArcade as well.

File: src/com/spikes2212/robot/commands/autonomous/MoveTurnRightAndDropGearAuto.java [hunk 3]
```diff
@@ -17,17 +18,21 @@
...
-	public static final Supplier<Double> movingTime = ConstantHandler
+			.addConstantDouble("MoveTurnRightAndDropGearAuto-turningSpeed", 0.2);
+	public static final Supplier<Double> movingStraightTime = ConstantHandler
 			.addConstantDouble("MoveTurnRightAndDropGearAuto-movingTime", 4);
 	public static final Supplier<Double> moveToGearTime = ConstantHandler
-			.addConstantDouble("MoveTurnLeftAndDropGearAuto-moveToGearTime", 1);
+			.addConstantDouble("MoveTurnRightAndDropGearAuto-moveToGearTime", 1);
 
 	public MoveTurnRightAndDropGearAuto() {
 		addSequential(new DriveTank(Robot.drivetrain, MoveStraightAndDropGearAuto.movingSpeed,
```
Comment (TomerSitton) [inline src/com/spikes2212/robot/commands/autonomous/MoveTurnRightAndDropGearAuto.java]: make driveArcade

**Comment by ThinkRedstone** [comment]
Closing this so it stops showing up under my pull requests

**Pull request closed without merging**